interfaces/Portalicious/src/app/pages/project-payments/components/create-payment/create-payment.component.ts

```diff
@@ -286,6 +286,11 @@ export class CreatePaymentComponent {
       return [];
     }
 
+    const totalPaymentAmount =
+      this.paymentAmount() * dryRunResult.sumPaymentAmountMultiplier;
+    const totalPaymentAmountFormat =
+      totalPaymentAmount % 1 === 0 ? '1.0-0' : '1.2-2';
```

Comment:
@arsforza Would it also work with just the format: `1.0-2`

So "minimally 0 decimal-digits, maximum 2", then it would be doing the "check if necessary to show `00` for you?

Reply:
For "precise money values" which is a term I just came up with, I actually think it would be wise to always show two decimal digits regardless (to avoid any confusion for the user).

Reply:
Yeah, I would also be all for "more explicit, less wrong" :D

But I hope these things where already considered in the design and the "shortest number possible with least confusing zeros" was the goal here.

Reply:
@elwinschmitz `1.0-2` shows `.5` instead of `.50`

Reply:
> But I hope these things where already considered in the design

Maybe wishful thinking, maybe design was ignored (at least I am not sure that I ever took the design as leading when copy-pasting the currency pipe code 🙈 )